flakey-test-tracking/src/jest/utils.js

```diff
@@ -0,0 +1,52 @@
+const { fmtFlakeIssue, fmtTestTitles } = require('../utils')
+
+const buildFlakeyDescribe = (describeBlock, flakeMap) => {
+  for (const child of describeBlock.children) {
+    switch (child.type) {
+      case 'describeBlock': {
+        buildFlakeyDescribe(child, flakeMap)
+        break
+      }
+      case 'test': {
+        const testID = getTestID(child)
+        const flakeErrors = flakeMap.get(testID)
+        if (flakeErrors !== undefined) {
+          child.errors = flakeErrors
+          child.status = 'flakey'
+        }
+        break
+      }
+    }
+  }
+  return describeBlock
+}
+
+const getTestID = (test) => {
```

Comment:
Ok, just to check.
This gets all the "descriptions" + "it" AND the path of the test, right?

Reply:
So the testID actually ends up being 'jobName -> packageName -> rootDescribeTitle -> ... -> testTitle'